examples/federation-workshop/cluster-1/bundle/ConfigMap.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
These ConfigMaps are not strictly necessary. 

They are just a backup measure to create the initial federation if the `ClusterFederatedTrustDomain` custom resource does not work for a reason.

That can be a life-saver during a live demo.